cfme/tests/services/test_dialog_regex_validation_in_catalog.py

```diff
@@ -169,4 +168,54 @@ def test_regex_dialog_validation_error():
             5.
             6. It shouldn't gives the validation error
     """
-    pass
+    service_dialog = appliance.collections.service_dialogs
+    element_data = {
+        "element_information": {
+            "ele_label": fauxfactory.gen_alphanumeric(15, start="ele_label_"),
+            "ele_name": fauxfactory.gen_alphanumeric(15, start="ele_name_"),
+            "ele_desc": fauxfactory.gen_alphanumeric(15, start="ele_desc_"),
+            "choose_type": "Text Box",
+        },
+        "options": {"validation_switch": True, "validation": "^[0-9]*$"},
+    }
+    sd = service_dialog.create(
+        label=fauxfactory.gen_alphanumeric(start="dialog_"), description="my dialog"
+    )
+    tab = sd.tabs.create(
+        tab_label=fauxfactory.gen_alphanumeric(start="tab_"), tab_desc="my tab desc"
+    )
+    box = tab.boxes.create(
+        box_label=fauxfactory.gen_alphanumeric(start="box_"), box_desc="my box desc"
+    )
+    box.elements.create(element_data=[element_data])
+
+    navigate_to(sd, "Edit")
+    view = appliance.browser.create_view(EditElementView)
```

Comment:
Q: is is not possible to navigate against the `element` object directly?

Reply:
No, after reaching to edit view page, we navigate to EditElement page to edit the element , I think we have to navigate by EditElementView here.